controllers/api.controller.js

```diff
@@ -81,3 +82,25 @@ exports.postCommentByArticle = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.patchVoteByArticle = async (req, res, next) => {
+  try {
+    const { article_id: articleId } = req.params;
+    if (isNaN(articleId)) {
```

Comment:
Custom error is fine here but you will also get an error from Postgres if you try to patch with NaN